app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :users
+
+  has_many :comments
+  has_many :likes
+
+  def recent_comments
+    comments.order('created_at Desc').limit(5)
+  end
+
+  after_save :update_post_counter
+
+  def update_post_counter
+    author.increment!(:posts_counter)
+  end
```

Comment:
- Since you have no author foreign_key, this would give an error in your code, please use `user` instead of `author` here. Please fix this 🙏 
- Also, you do not have a `post_counter` attribute in your User model, please use the appropriate attribute 🙏